src/databricks/labs/ucx/runtime.py

```diff
@@ -23,6 +23,127 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(cfg: WorkspaceConfig) -> str:
+    return f"""vw_failure_summary AS WITH failuretab (failures, component) AS (
+        SELECT
+          failures,
+          "jobs" AS component
```

Comment:
```suggestion
          "jobs" AS object_type
```
Please call it object type everywhere